middle_end/flambda2/from_lambda/closure_conversion.ml

```diff
@@ -736,9 +738,9 @@ let close_let acc env id user_visible defining_expr
             (Env.add_block_approximation body_env (Name.var var) approxs
                alloc_mode)
         | Prim (Binary (Block_load _, block, field), _) -> (
-          match Env.find_value_approximation body_env block with
+          match Env.find_value_approximation env block with
```

Comment:
I don't think it's actually wrong, but isn't that just a leftover from the branch based on #776 ?

Reply:
Oh right, it is. Well spotted, I'll fix it.